server/services/v1/auth/gotrue.go

```diff
@@ -550,7 +616,7 @@ func _listAppKeys(ctx context.Context, g *gotrue, project string) ([]*appKeyInte
 			// parse string time to millis using rfc3339 format
 			createdAtMillis = readDate(createdAtStr)
 		}
-
+		log.Info().Interface("metadata", appMetadata).Msg("AppMetadata")
```

Comment:
Is this debug message?

Reply:
yes it was left out by mistake. removed it.